Example/Stripe iOS Example (Custom)/BrowseExamplesViewController.m

```diff
@@ -0,0 +1,157 @@
+    return 4;
+}
+
+- (UITableViewCell *)tableView:(UITableView *)tableView cellForRowAtIndexPath:(NSIndexPath *)indexPath {
+    UITableViewCell *cell = [UITableViewCell new];
+    switch (indexPath.row) {
+        case 0:
+            cell.textLabel.text = @"Card";
+            break;
+        case 1:
+            cell.textLabel.text = @"Card + 3DS";
+            break;
+        case 2:
+            cell.textLabel.text = @"Apple Pay";
+            break;
+        case 3:
+            cell.textLabel.text = @"Sofort";
+            break;
+    }
+    return cell;
+}
+
+- (void)tableView:(UITableView *)tableView didSelectRowAtIndexPath:(NSIndexPath *)indexPath {
+    UIViewController *viewController;
+    switch (indexPath.row) {
+        case 0: {
+            CardExampleViewController *exampleVC = [CardExampleViewController new];
+            exampleVC.delegate = self;
+            viewController = exampleVC;
+            break;
+        }
+        case 1: {
+            ThreeDSExampleViewController *exampleVC = [ThreeDSExampleViewController new];
+            exampleVC.delegate = self;
+            viewController = exampleVC;
+            break;
+        }
+        case 2: {
+            ApplePayExampleViewController *exampleVC = [ApplePayExampleViewController new];
+            exampleVC.delegate = self;
+            viewController = exampleVC;
+            break;
+        }
+        case 3: {
+            SofortExampleViewController *exampleVC = [SofortExampleViewController new];
+            exampleVC.delegate = self;
+            viewController = exampleVC;
+            break;
+        }
+    }
+    [self.navigationController pushViewController:viewController animated:YES];
+}
+
+#pragma mark - STPBackendCharging
+
+- (void)createBackendChargeWithSource:(NSString *)sourceID completion:(STPSourceSubmissionHandler)completion {
+    if (!BackendBaseURL) {
+        NSError *error = [NSError errorWithDomain:StripeDomain
+                                             code:STPInvalidRequestError
+                                         userInfo:@{NSLocalizedDescriptionKey: @"You must set a backend base URL in Constants.m to create a charge."}];
+        completion(STPBackendChargeResultFailure, error);
+        return;
+    }
+
+    // This passes the token off to our payment backend, which will then actually complete charging the card using your Stripe account's secret key
+    NSURLSessionConfiguration *config = [NSURLSessionConfiguration defaultSessionConfiguration];
+    NSURLSession *session = [NSURLSession sessionWithConfiguration:config];
+
+    NSString *urlString = [BackendBaseURL stringByAppendingPathComponent:@"create_charge"];
+    NSURL *url = [NSURL URLWithString:urlString];
+    NSMutableURLRequest *request = [[NSMutableURLRequest alloc] initWithURL:url];
+    request.HTTPMethod = @"POST";
+    NSString *postBody = [NSString stringWithFormat:@"source=%@&amount=%@", sourceID, @1099];
```

Comment:
I think it's fine for now – it would be more secure to create an order and pass up the order ID, but that would complicate the example, and I think that's out of the scope of this change (we'd have to update the swift example too).